src/components/Sorting/sorting.jsx

```diff
@@ -1,14 +1,33 @@
 import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { getMovies, setSortParam } from '../../redux/movies/actions';
+import { getCurrentRequest, getCurrentFilterParam } from '../../redux/movies/selectors';
+import { sortingItems } from '../../mockedData';
 
 import './sorting.scss';
 
-import { sortingItems } from '../../mockedData';
+const dictionary = {
```

Comment:
`dictionary` does not have much sense. `sortParams` will be better

Reply:
fixed